src/ontology/imports/foodon_product_import.owl

```diff
@@ -33599,6 +33605,98 @@ Fishcakes as defined in the Oxford Dictionary of Food and Nutrition are chopped
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004167 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004167">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00004166"/>
+        <obo:IAO_0000114 rdf:resource="http://purl.obolibrary.org/obo/IAO_0000428"/>
+        <obo:IAO_0000115 xml:lang="en">A cut of beef and part of the sub-prime cut known as the chuck.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119>https://rollybrook.com/lexicon-groups.htm</obo:IAO_0000119>
+        <obo:IAO_0000119>https://www.mexconnect.com/articles/2398-choice-cut-or-mystery-meat-a-guide-to-mexican-butcher-shops-part-i-beef/</obo:IAO_0000119>
+        <rdfs:label xml:lang="en">diezmillo</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004168 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004168">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03317170"/>
+        <obo:IAO_0000115 xml:lang="en">The process of slow cooking meat to make it easier to chew or cut.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://en.wikipedia.org/wiki/Meat_tenderness</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-02-04T22:11:53Z</dc:date>
+        <rdfs:label xml:lang="en">meat (tenderized)</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004169 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004169">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00001180"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00002674"/>
+        <obo:IAO_0000115 xml:lang="en">A chicken wing which has been prepared.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-04T22:15:34Z</dc:date>
+        <oboInOwl:hasNarrowSynonym xml:lang="en">party wing</oboInOwl:hasNarrowSynonym>
+        <rdfs:label xml:lang="en">prepared chicken wing</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00004171 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00004171">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003357"/>
+        <obo:IAO_0000115 xml:lang="en">A pet treat which is small in size.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2022-01-05T11:00:23Z</dc:date>
+        <oboInOwl:hasNarrowSynonym xml:lang="en">mini-bite treat</oboInOwl:hasNarrowSynonym>
```

Comment:
Ugh, sorry - I think I made a mistake in saying mini-bite treat and petite treat are narrow synonyms. I think they are exact synonyms (where both have hidden context of pet treats).